docs/VtoStatuses.md

```diff
@@ -1,78 +1,41 @@
 ## Vehicle Transfer Order Statuses
 
-The `VehicleTransferOrder.status` field has the following possible values:
-```
-DRAFT
-READY
-AVAILABLE
-CLAIMED
-UNCLAIMED
-APPROVED
-ASSIGNED
-REJECTED
-ACCEPTED
-DRIVER_ARRIVED
-PICKED_UP
-SIGNATURE_ON_PICKUP
-PROVIDED_ETA_DROPOFF
-UNLOADED
-DELIVERED
-CANCELED
-COMPLETE
-ERROR
-```
-
-### Draft
-The Vehicle Transfer Order has been successfully received by the system and is being processed.
-
-### Ready
-The Vehicle Transfer Order has been successfully processed and will be made available to be claimed shortly.
-
-### Available
-The Vehicle Transfer Order has been successfully entered into the system and is available to be accepted by a
- transporter.
-
-### Claimed
-The Vehicle Transfer Order has been claimed by a transportation company to be transferred.
-
-### Unclaimed
-The Vehicle Transfer Order has been unclaimed by a transportation company.
-
-### Approved
-The Vehicle Transfer Order has been reviewed and approved by RunBuggy after transportation company unclaimed due to issues with order.
-
-### Assigned
-The Vehicle Transfer Order has been assigned to a transportation company driver.
-
-### Rejected
-The Vehicle Transfer Order has been rejected by a transportation company driver.
-
-### Accepted
-The Vehicle Transfer Order has been accepted by a transportation company driver.
+The following events are vehicleTranferOrder(VTO) event statuses that are sent to the shipper customer's endpoint on the RunBuggy webhook.  One event is received for each VTO within an order, and each vehicle on an order has a VTO.  Because there can be multiple VTOs and events received per order, it's recommended to use the VTO ID rather than the order ID to reference each event with the appropriate record in your system.
```

Comment:
This needs improved or removed. 

Let's not use "vehicleTranferOrder". "Vehicle Transfer Order" is more readable.

These statuses are not just used for webhooks. So this does not make sense as the intro the doc. These are the status changes that a VTO goes through and ALSO used for webhooks. We have a dedicated guide for webhooks. Please link out to that guide instead of going into webhook details here which is simply a guide to statuses.

Reply:
Also, why did we change the format of this page? In some cases there is less info and I find the changes confusing.

Breaking out into Transporter Update Events and Exception Events for example. This isn't a guide to events but VTO statuses. And some statuses lost their description altogether e.g. "ERROR"